File: api/src/main/kotlin/org/team619/rutor/model/Comment.java
package org.team619.rutor.model;

import org.team619.rutor.model.core.DefaultEntity;

/**
 * Created by BORIS on 02.09.2016.
 */
public class Comment implements DefaultEntity {

    private final String id;
    private final String author;
    private final String created;
    private final String stars;
    private final String body;

    public Comment(String id, String author, String created, String stars, String body) {
        this.id = id;
        this.author = author;
        this.created = created;
        this.stars = stars;
        this.body = body;
    }

    public String getBody() {
        return body;
    }

    public String getCreated() {
        return created;
    }

    public String getId() {
        return id;
    }

    public String getStars() {
        return stars;
    }

    public String getAuthor() {
        return author;
    }
}

File: api/src/main/kotlin/org/team619/rutor/model/Comment.kt
package org.team619.rutor.model

import org.team619.rutor.core.Page

/**
 * Created by BORIS on 02.09.2016.
 */
data class Comment(
        val id: String,
        val author: String,
        val created: String,
        val stars: String,
        val body: String) : Page
